### ISSUE TITLE
Feature request: Add syllabification support

### ISSUE THREAD

**Issue by ftyers** [ftyers/commonvoice-utils#46]
I would like to be able to count syllables and segment words by syllables, e.g. in the word:

caltlamachtiloyan → cal·tla·mach·til·oy·an
camioneta → ca·mio·ne·ta

**Comment by HarikalarKutusu** [comment]
I guess each language will have a different code for it. I found a couple of algorithms for Turkish, all seem to have problems.

**Comment by ftyers** [comment]
Yep,  I guess it can be done with FST, there might also be code in LaTeX that can be adapted.
